index.js

```diff
@@ -261,6 +261,10 @@ module.exports = {
             // new-cap throws errors with property decorators
             'new-cap': 'off',
 
+            // TypeScript will be parsed in strict mode and output the `use-stict`
+            // directive for the transpiled JavaScript automatically.
+            'strict': 'off',
```

Comment:
Do we want to turn the rule off, or do we want to actively prevent `"use strict"` statements since they are redundant?